vllm/_custom_ops.py

```diff
@@ -10,6 +12,15 @@
     logger = init_logger(__name__)
     logger.warning("Failed to import from vllm._C with %r", e)
 
+if is_xpu():
+    try:
+        from vllm._ipex_ops import ipex_cache_ops as vllm_cache_ops
+        from vllm._ipex_ops import ipex_ops as vllm_ops
+    except ImportError as e:
+        from vllm.logger import init_logger
+        logger = init_logger(__name__)
+        logger.warning("Import error msg: %s", e.msg)
```

Comment:
Should we raise an error instead?

Reply:
I have reverted this file back since xpu path will not use compiled kernels.